doc/sphinx/changes.rst

```diff
@@ -8,6 +8,675 @@ Recent changes
+  A bug producing ill-typed instances of existential variables when let-ins
+  interleaved with assumptions
+  (`#13387 <https://github.com/coq/coq/pull/13387>`_,
+  fixes `#12348 <https://github.com/coq/coq/issues/13387>`_,
+  by Hugo Herbelin).
+
+  .. _813Notations:
+
+Notations
+^^^^^^^^^
+
+- **Changed:**
+  In notations (except in custom entries), the misleading :n:`@syntax_modifier`
+  :n:`@ident ident` (which accepted either an identifier or
+  a :g:`_`) is deprecated and should be replaced by :n:`@ident name`. If
+  the intent was really to only parse identifiers, this will
+  eventually become possible, but only as of Coq 8.15.
+  In custom entries, the meaning of :n:`@ident ident` is silently changed
+  from parsing identifiers or :g:`_` to parsing only identifiers
+  without warning, but this presumably affects only rare, recent and
+  relatively experimental code
+  (`#11841 <https://github.com/coq/coq/pull/11841>`_,
+  fixes `#9514 <https://github.com/coq/coq/pull/9514>`_,
+  by Hugo Herbelin).
+- **Added:**
+  :flag:`Printing Float` flag to print primitive floats as hexadecimal
+  instead of decimal values. This is included in the :flag:`Printing All` flag
+  (`#11986 <https://github.com/coq/coq/pull/11986>`_,
+  by Pierre Roux).
+- **Changed:**
+  Improved support for notations/abbreviations with mixed terms and patterns (such as the forcing modality)
+  (`#12099 <https://github.com/coq/coq/pull/12099>`_,
+  by Hugo Herbelin).
+- **Deprecated**
+  ``Numeral.v`` is deprecated, please use ``Number.v`` instead.
+  (`#12218 <https://github.com/coq/coq/pull/12218>`_,
+  by Pierre Roux).
+- **Changed**
+  Rational and real constants are parsed differently.
+  The exponent is now encoded separately from the fractional part
+  using ``Z.pow_pos``. This way, parsing large exponents can no longer
+  blow up and constants are printed in a form closer to the one in which they
+  were parsed (i.e., ``102e-2`` is reprinted as such and not ``1.02``).
+  (`#12218 <https://github.com/coq/coq/pull/12218>`_,
+  by Pierre Roux).
+- **Removed**
+  OCaml parser and printer for real constants have been removed.
+  Real constants are now handled with proven Coq code.
+  (`#12218 <https://github.com/coq/coq/pull/12218>`_,
+  by Pierre Roux).
+- **Added:**
+  :ref:`Number Notation <number-notations>` and :ref:`String Notation
+  <string-notations>` commands now
+  support parameterized inductive and non inductive types
+  (`#12218 <https://github.com/coq/coq/pull/12218>`_,
+  fixes `#12035 <https://github.com/coq/coq/issues/12035>`_,
+  by Pierre Roux, review by Jason Gross and Jim Fehrle for the
+  reference manual).
+- **Changed:**
+  Scope information is propagated in indirect applications to a
+  reference prefixed with :g:`@@`; this covers for instance the case
+  :g:`r.(@@p) t` where scope information from :g:`p` is now taken into
+  account for interpreting :g:`t` (`#12685
+  <https://github.com/coq/coq/pull/12685>`_, by Hugo Herbelin).
```

Comment:
It looks like these double `@` were an attempt to escape the `@`, but the ambiguity of `@` only occurs in `:n:` contexts, so these should be made single `@`:
```suggestion
- **Changed:**
  Scope information is propagated in indirect applications to a
  reference prefixed with :g:`@`; this covers for instance the case
  :g:`r.(@p) t` where scope information from :g:`p` is now taken into
  account for interpreting :g:`t` (`#12685
  <https://github.com/coq/coq/pull/12685>`_, by Hugo Herbelin).
```